src/phoenix/server/api/types/PromptVersion.py

```diff
@@ -59,6 +59,29 @@ async def user(self, info: Info[Context, None]) -> Optional[User]:
             user = await session.get(models.User, self.user_id)
         return to_gql_user(user) if user is not None else None
 
+    @strawberry.field
+    async def previous_version(self, info: Info[Context, None]) -> Optional["PromptVersion"]:
+        async with info.context.db() as session:
+            current_version = await session.get(models.PromptVersion, self.id_attr)
+            if current_version is None:
+                return None
+
+            prompt_id = current_version.prompt_id
+            created_at = current_version.created_at
+
+            stmt = (
+                select(models.PromptVersion)
+                .where(models.PromptVersion.prompt_id == prompt_id)
+                .where(models.PromptVersion.created_at < created_at)
+                .order_by(models.PromptVersion.created_at.desc())
```

Comment:
nit. the primary key would be more strictly ordered
```suggestion
                .where(models.PromptVersion.id < self.id_attr)
                .order_by(models.PromptVersion.id.desc())
```